### ISSUE TITLE
[Not an issue] Vertex?

### ISSUE THREAD

**Issue by RaitaroH** [erikdubois/Arc-Theme-Colora-Collection#1]
It is possible to use your scripts (most likely requires some tweaking) for the Vertex theme also made by the same guy? That will be fantastic.

**Comment by erikdubois** [comment]
Do try and let me know

Sent from my iPad, embrace the typos and the short answers

**Comment by RaitaroH** [comment]
So I used your scripts. Here is what I did.

0. I am using Linux Mint 17.3 so you know.

1. Remove the vertex theme
sudo rm -rf /usr/share/themes/{Vertex,Vertex-Dark,Vertex-Light,Vertex-Gnome-Shell,Vertex-Gnome-Shell-3.16,Vertex-Cinnamon}
rm -rf ~/.local/share/themes/{Vertex,Vertex-Dark,Vertex-Light,Vertex-Gnome-Shell,Vertex-Gnome-Shell-3.16,Vertex-Cinnamon}
rm -rf ~/.themes/{Vertex,Vertex-Dark,Vertex-Light,Vertex-Gnome-Shell,Vertex-Gnome-Shell-3.16,Vertex-Cinnamon}

2. Cloned the vertex theme in my home:
git clone https://github.com/horst3180/vertex-theme

3. Cloned your code
git clone https://github.com/erikdubois/arc-theme-colora

4. Copy pasted the scripts as you said in the vertex folder

5. Edited the scripts:

In 1-change-colors.sh replace:
	5294e2 with 4080fb
I also changed the new color to mine: 00adee (if you check my themes out you know why :))  )

In the other scripts replace:
	Arc with Vertex
	Darker with Light (it may not be the best choice but... )

6. I executed the scripts in order (damn the render took so long). I got this at the end:

making the assets of xfwm4

sh: 0: Can't open render-assets.sh
All assets have been made.
Svg's have been exported from the assets svg files
and converted to png's

Also I looked inside the theme folders (after executing script 4) and it seems that the icons did not change. For example in Vertex-Dark/gtk-2.0/Check-Radio the checked button is the same. In gtk3 also the same (maybe because is not the same as in Arc). Also in Nemo the border around the path of the folder (like the url-bar) did not change either. In audacious the selected background of a song did not change. The selected background in menus didn't. So is with vlc.

What did change are the selected background on menus such as the context menu on desktop and folders. The background color when I highlight something. Not much really.

7. Script number 5 didn't actually work though. I just let them be in the /usr/share/themes because it is better for me like that.


I think the the scripts will work but they do need some tweaking though. It will be awesome if you did take the time to make another collection of themes based on Vertex. 

Thank you.

**Comment by RaitaroH** [comment]
TL;DR
It didn't work. It had the same effect of me going in the theme folders, find all the code bits and replace 4080fb with 00adee. That is it.

So I used your scripts. Here is what I did.

0. I am using Linux Mint 17.3 so you know.

1. Remove the vertex theme
sudo rm -rf /usr/share/themes/{Vertex,Vertex-Dark,Vertex-Light,Vertex-Gnome-Shell,Vertex-Gnome-Shell-3.16,Vertex-Cinnamon}
rm -rf ~/.local/share/themes/{Vertex,Vertex-Dark,Vertex-Light,Vertex-Gnome-Shell,Vertex-Gnome-Shell-3.16,Vertex-Cinnamon}
rm -rf ~/.themes/{Vertex,Vertex-Dark,Vertex-Light,Vertex-Gnome-Shell,Vertex-Gnome-Shell-3.16,Vertex-Cinnamon}

2. Cloned the vertex theme in my home:
git clone https://github.com/horst3180/vertex-theme

3. Cloned your code
git clone https://github.com/erikdubois/arc-theme-colora

4. Copy pasted the scripts as you said in the vertex folder

5. Edited the scripts:

In 1-change-colors.sh replace:
	5294e2 with 4080fb
I also changed the new color to mine: 00adee (if you check my themes out you know why :))  )

In the other scripts replace:
	Arc with Vertex
	Darker with Light (it may not be the best choice but... )

6. I executed the scripts in order (damn the render took so long). I got this at the end:

making the assets of xfwm4

sh: 0: Can't open render-assets.sh
All assets have been made.
Svg's have been exported from the assets svg files
and converted to png's

Also I looked inside the theme folders (after executing script 4) and it seems that the icons did not change. For example in Vertex-Dark/gtk-2.0/Check-Radio the checked button is the same. In gtk3 also the same (maybe because is not the same as in Arc). Also in Nemo the border around the path of the folder (like the url-bar) did not change either. In audacious the selected background of a song did not change. The selected background in menus didn't. So is with vlc.

What did change are the selected background on menus such as the context menu on desktop and folders. The background color when I highlight something. Not much really.

7. Script number 5 didn't actually work though. I just let them be in the /usr/share/themes because it is better for me like that.


I think the the scripts will work but they do need some tweaking though. It will be awesome if you did take the time to make another collection of themes based on Vertex. 

Thank you.

**Comment by erikdubois** [comment]
not a priority right now
i will keep this idea in the back of my head though
